packages/jest-reporters/src/GitHubActionsReporter.ts

```diff
@@ -26,10 +33,69 @@ type AnnotationOptions = {
 
 const titleSeparator = ' \u203A ';
 
+type PerformanceInfo = {
+  end: number;
+  runtime: number;
+  slow: boolean;
+  start: number;
+};
+
+type ResultTreeLeaf = {
+  name: string;
+  passed: boolean;
+  duration: number;
+  children: Array<never>;
+};
+
+type ResultTreeNode = {
+  name: string;
+  passed: boolean;
+  children: Array<ResultTreeNode | ResultTreeLeaf>;
+};
+
+type ResultTree = {
+  children: Array<ResultTreeLeaf | ResultTreeNode>;
+  name: string;
+  passed: boolean;
+  performanceInfo: PerformanceInfo;
+};
+
 export default class GitHubActionsReporter extends BaseReporter {
   static readonly filename = __filename;
+  private readonly silent: boolean;
+
+  constructor(
+    _globalConfig: Config.GlobalConfig,
+    reporterOptions: Record<string, unknown> = {silent: false},
+  ) {
+    super();
+    const silentOption = reporterOptions.silent;
+    if (silentOption !== null && silentOption !== undefined) {
+      this.silent = silentOption as boolean;
+    } else {
+      this.silent = false;
+    }
```

Comment:
Is there a need to normalize this option?

```suggestion
  constructor(
    _globalConfig: Config.GlobalConfig,
    private options: {silent?: boolean},
  ) {
    super();
```